app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,100 @@
+TASKS = ["be cool", "be calm", "be collected", "lose your ever-loving mind"]
+
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+
+  def show
+    task_id = params[:id]
+    @task = Task.find_by(id: task_id)
+    if @task.nil?
+      redirect_to tasks_path
+      return
+      head :not_found
+      return
+    end
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    @task = Task.new(
+        name: params[:task][:name],
+        description: params[:task][:description],
+        completed_at: params[:task][:completed_at])
```

Comment:
It wasn't required for this project, but I encourage you to use a strong params method in the future! You already have one defined below, it'd be best to refactor this line to use it :)